src/main/java/com/github/bibsysdev/nva/cristin/person/PersonsWrapper.java

```diff
@@ -0,0 +1,93 @@
+package com.github.bibsysdev.nva.cristin.person;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.ALWAYS;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.github.bibsysdev.nva.cristin.person.model.nva.NvaPerson;
+import java.net.URI;
+import java.util.List;
+import nva.commons.core.JacocoGenerated;
+
+@JsonInclude(ALWAYS)
+@JsonPropertyOrder("@context")
```

Comment:
You already define this string in a variable CONTEXT. Why not reuse that instead of writing the string two times? Not very important for now though. I think you can use it on the class level since it it declared final

Reply:
Done.